@here/harp-mapview/lib/Utils.ts

```diff
@@ -686,6 +686,21 @@ export namespace MapViewUtils {
         return ((mapView.focalLength * tileSize) / 256) * Math.cos(cameraPitch);
     }
 
+    /**
+     * Calculates the zoom level, which corresponds to the current distance from
+     * camera to the ground.
+     *
+     * @param mapView Instance of MapView.
+     * @param groundDistance Distance from the ground to the camera
+     */
+    export function calculateZoomLevelFromDistanceToGround(
```

Comment:
Please add some tests that shows that going from zoomlevel to distance and back to zoomlevel actually works